rfb/rfb.h

```diff
@@ -457,6 +459,9 @@ typedef struct _rfbClientRec {
     void* clientData;
     ClientGoneHookPtr clientGoneHook;
 
+    /** clientFURHook is called when a client requests a frame buffer update. */
+    ClientFURHookPtr clientFURHook;
```

Comment:
Please 
* use the full name, no the acronym
* put the field at the end of the struct to retain ABI compaibility

Reply:
Updated this commit to address your comments.
Thanks!